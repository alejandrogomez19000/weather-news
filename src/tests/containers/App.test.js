import React from "react";
import App from "App";
import ShallowRender from "react-test-renderer/shallow";

const renderer = new ShallowRender();

describe("render app", () => {
  it("should be render", async () => {
    renderer.render(<App />);
    const renderOutput = renderer.getRenderOutput();
    expect(renderOutput).toMatchSnapshot();
  });
});
