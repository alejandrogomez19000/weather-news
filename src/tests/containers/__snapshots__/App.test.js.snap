// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app should be render 1`] = `<Routes />`;
